src/vsr.zig

```diff
@@ -1055,11 +1066,79 @@ pub fn sector_ceil(offset: u64) u64 {
 }
 
 pub fn checksum(source: []const u8) u128 {
-    @setEvalBranchQuota(4000);
+    // From std.crypto.aegis.State128L:
```

Comment:
Personal preference, so I'm happy to go either way, but I generally think it's nicer to keep backported code like this in a separate file.